packages/components/src/jetpack-upsell-card/stories/index.stories.tsx

```diff
@@ -10,17 +25,29 @@ const upgradeUrls = {
 	social: 'https://jetpack.com',
 	video: 'https://jetpack.com',
 };
-const UpsellSection = ( props: any ) => (
-	<div className="jetpack-upsell-section-story" style={ { margin: 'auto', maxWidth: '1000px' } }>
-		<JetpackUpsellCard purchasedProducts={ [] } upgradeUrls={ upgradeUrls } { ...props } />
-	</div>
-);
 
-export const Default = () => <UpsellSection />;
```

Comment:
This doesn't pass through props. Props must be passed through correctly for Controls to work.